### ISSUE TITLE
Create component - Switch

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-flutter#49]
# Description
**As a** user of the Ouds library
**I want** to be able to use the component
**So that** I can add an Ouds component in my own projects

**As a** user of the library
**I want** to be able to use the component
**So that** I have access to the documentation on how to use it

**As a** user of the Demo App
**I want** to access the Component 
**So that** I can understand how to use it:
* while using the demo app I can interact with the available options and observe the changes on the component itself
* I can see the code associated to the component customization
* I can copy and paste the code from the app based on the selected customization
* or when looking into the code of the app itself, I would also understand how to use it

# Context 
The component is described in the Design tools and allows to describe the various states and variants.
## Variants
TBD

## Demo App
TBD

**Comment by B3nz01d** [comment]
@Tayebsed93  I found the following issues:
* in the Switch item the switch is defaulted on the left and it should be on the right
* when the text is very log both in the label and in the helper text the position of the switch and the icon are not in the middle (between top and bottom) - it's suppose to be one third up
* you do not use the switch withi the customization bottom sheets of all the components.

**Comment by B3nz01d** [comment]
@MaximeTonnerre  can you provide the maxheight for this positioning?

**Comment by Tayebsed93** [comment]
> [@MaximeTonnerre](https://github.com/MaximeTonnerre) can you provide the maxheight for this positioning?

@B3nz01d The token you're referring to is:

`ouds/💠_input/control-item/size/max-height-assets-container`

The precise positioning is set at 96

**Comment by Tayebsed93** [comment]
A new version has been submitted. Kindly verify the version number mentioned in the PR comments : https://github.com/Orange-OpenSource/ouds-flutter/pull/196

**Comment by B3nz01d** [comment]
@Tayebsed93 & @AhmedAmineZr 
* in the current version build 261 the reversed selector is set correctly, but the default state shown to the user is with the reversed option unselected
* In the ios version the when you put a large amount of text you do not get the full information showing. (this does not happen on android)
![image](https://github.com/user-attachments/assets/6c1c9889-813e-4527-a635-b101ed8347ae)

![image](https://github.com/user-attachments/assets/3bdfe652-51ae-4d17-9383-7e2bc014453c)

* also for some reason the helper text on ios sometimes does not get shown when you reopen the customization bottom sheet

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (beta) (290)
Version: 0.4.0
Build type: BETA
Build details (GitHub): 49
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (beta) (290)
Version: 0.4.0
Build type: BETA
Build details (GitHub): 49
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06  (dev team)
cc @jerome-regnier  (design team)

**Comment by pya35** [comment]
see:
- [ ] #364

**Comment by B3nz01d** [comment]
Issue closed during Sprint # 20

**Comment by Tayebsed93** [comment]
I added the label “OK Accessibility” because @pya35 previously created a bug ticket (#364) about this issue, and later marked it as resolved.
So I assume they simply forgot to update the main ticket accordingly, and that everything is actually fine.
